### DISCUSSION TITLE
Non-Racked Devices on (v4.0.8), removed or moved?

### DISCUSSION THREAD

**Discussion by redstarsachin** [netbox-community/netbox#17102] [Help Wanted!] [answered]
Non-Racked Devices on (v4.0.8), is it removed? 
couldn't find!

**Comment by chris240189** [comment, marked as answer]
when looking at a rack there is a tab for non-racked devices
![image](https://github.com/user-attachments/assets/25e134cc-1ac3-4733-b357-629451037bd2)

**Reply by redstarsachin** [reply]
Thanks!
